README.md

```diff
@@ -15,7 +15,8 @@ Our Kount-PHP-SDK Page includes all the documentation related to PHP SDK. Everyt
 * Now we have a Github Page, that helps you find all helpful resources building applications using Kount-PHP-SDK.
 
 ## Prerequisities
-* PHP 5.5 or above
+* PHP 7.2 or above
```

Comment:
I see in composer.json we require 7.2.5, but the docs reference 7.2.  Should we be more specific in the docs, to match the specificity in composer.json?

Reply:
Yes, we will do that.